packages/compass-aggregations/src/stores/store.ts

```diff
@@ -156,7 +159,7 @@ const configureStore = (options: ConfigureStoreOptions) => {
     options.dataProvider.dataProvider as DataService,
     initialPipelineSource
   );
-  const pipelineStorage = new PipelineStorage();
+  const pipelineStorage = new PipelineStorage(basepath);
```

Comment:
why is this a better alternative than having it inside the storages? if the reason is testing you can still have a default storage path and have the constructor accept a param to override it.

in this way you are giving a "passing knowledge" to `store.ts` of where the files are saved, which store.ts can't use directly.

Reply:
Yes, I did it for that reason (testability). I do see your point about sharing knowledge in `store.ts`, so I don't have a strong preference here.

Reply:
Yeah, btw having a way to change the base directory in the constructor is a great idea, we should do it! If you are ok with it, let's also keep the default in the storages so we make it both testable and we won't leak details in the client code.